consensus/hotstuff_leader.go

```diff
@@ -200,6 +201,17 @@ func (handler *HotstuffLeaderMessageHandler) HandleCommitMessage(m *consensusMod
 	// There is no "replica behavior" to imitate here
 
 	m.paceMaker.NewHeight()
+
+	m.
+		GetBus().
+		GetTelemetryModule().
+		IncCounter("consensus_blockchain_height")
```

Comment:
IIRC this should be gauge, not a counter, because we can reset the node to 0 height. Counters can only go up, however.